x/bank/client/rest/sendtx.go

```diff
@@ -54,13 +54,20 @@ func SendRequestHandlerFn(cdc *codec.Codec, kb keys.Keybase, cliCtx context.CLIC
 			return
 		}
 
-		info, err := kb.Get(req.BaseReq.Name)
-		if err != nil {
-			utils.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
-			return
+		var from sdk.AccAddress
+		if req.BaseReq.GenerateOnly {
```

Comment:
So if I'm reading this right, on generate only it just adds a `From` address to the base req and then will `CompleteAndBroadcastTxREST` right? If so this would need some more modification.

Reply:
It takes the from address from base_req instead of resolving it from the username.

Reply:
Plus it doesn't complete and broadcast it yes. Probably needs more modification. Can someone take this over who is opinionated about the SDK code?